scripts/ci_setup.py

```diff
@@ -396,7 +393,7 @@ def main(args: Any):
 
     # When running on internal repos, the repository comes to us incorrectly
     # (ie https://github.com/dotnet-coreclr). Replace dashes with slashes in that case.
-    repo_url = None if use_core_sdk else args.repository.replace('-','/')
```

Comment:
I think it would make sense to either remove this completely or continue to use `use_core_sdk`. There are only a few other places that use `use_core_sdk`, I have a minor lean towards keeping `use_core_sdk`.